### PR TITLE
Unapologetic Dragon Buffs, Now With Language

### PR DESCRIPTION
**Pull request by Dhul-Qarnayn**
Adds dragons and Draconic. Hacked together from warforms and Azaziba, so likely problematic.

### CODE REVIEW

File: code/modules/mob/language/station.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		"ra", "ar", "re", "er", "ri", "ir", "ro", "or", "ru", "ur", "rs", "sr",
 		"a",  "a",  "e",  "e",  "i",  "i",  "o",  "o",  "u",  "u",  "s",  "s"
 	)
+	partial_understanding = list(LANGUAGE_KOBOLD = 50)
```
Comment (MarinaGryphon) [inline code/modules/mob/language/station.dm]:
```suggestion
	partial_understanding = list(LANGUAGE_DRAGON = 70)
```

File: code/modules/mob/language/station.dm [hunk 2]
```diff
@@ -41,4 +42,14 @@
 			return pick("exclaims","shouts","yells") //TODO: make the basic proc handle lists of verbs.
 		if("?")
 			return ask_verb
-	return speech_verb
\ No newline at end of file
+	return speech_verb
+
+/datum/language/dragon
+	name = LANGUAGE_DRAGON
+	desc = "An elaborate, high-context language spoken universally by dragons and widely by magicians. Regarded as the ideal form of language by most of the former two."
```
Comment (MarinaGryphon) [inline code/modules/mob/language/station.dm]:
```suggestion
	desc = "An elaborate, high-context language spoken universally by dragons and widely by magicians, regarded as the ideal form of language by most of the former."
```

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 3]
```diff
@@ -625,20 +625,20 @@ mob/living/carbon/human/proc/change_monitor()
 	if (victims.len)
 		admin_attacker_log_many_victims(src, victims, "used rebel yell to stun", "was stunned by [key_name(src)] using rebel yell", "used rebel yell to stun")
 
-/mob/living/carbon/human/proc/formic_spray()
+/mob/living/carbon/human/proc/fire_spray()
 	set category = "Abilities"
-	set name = "Napalm"
-	set desc = "Spew a cone of ignited napalm in front of you"
+	set name = "Searing Burst"
+	set desc = "Spew a cone of magical flame in front of you"
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/human_powers.dm]:
```suggestion
	set desc = "Spew a cone of magical flame in front of you."
```

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 4]
```diff
@@ -625,20 +625,20 @@ mob/living/carbon/human/proc/change_monitor()
...
+		to_chat(src,"<span class='notice'>You are too tired to spray fire!</span>")
 		return
 
 	if(stat || paralysis || stunned || weakened || lying || restrained() || buckled)
-		to_chat(src,"<span class='notice'>You cannot spray napalm in your current state.</span>")
+		to_chat(src,"<span class='notice'>You cannot spray fire in your current state.</span>")
 		return
 
-	last_special = world.time + 100
+	last_special = world.time + 20
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/human_powers.dm]: This is a *two second* cooldown for spraying fire.

Comment (MarinaGryphon) [reply code/modules/mob/living/carbon/human/human_powers.dm]:
```suggestion
	last_special = world.time + 20 SECONDS
```

Comment (Dhul-Qarnayn) [reply code/modules/mob/living/carbon/human/human_powers.dm]: Suggest to make it twenty seconds so I don't have to make a new commit, smh.

**Pull request merged**